apps/site/src/pages/api/login-with-login-code.api.ts

```diff
@@ -74,7 +84,28 @@ export default createBaseHandler<
         );
       }
 
-      await loginCode.setToUsed(db);
+      if (loginCode.variant === "linkWordpress") {
+        const { wordpressInstanceUrl } = loginCode;
+        if (!wordpressInstanceUrl) {
+          return res.status(500).json(
+            formatErrors({
+              msg: "Internal error. Please try again.",
+              param: "code",
+              value: code,
+            }),
+          );
+        }
+
+        await loginCode.setToUsed(db);
+
+        await user.addWordpressInstanceUrlAndVerify(
+          db,
+          wordpressInstanceUrl,
+          !user.hasVerifiedEmail,
```

Comment:
Shouldn't this always be `updateReferrer = false` for these `login` request?
This is currently updating the referrer on every `linkWordpress` login as far as I can tell (which I suppose should have happened in the sign up step?)

Do we have any calls to `addWordpressInstanceUrlAndVerify` that are ever `false`?

Reply:
Yes, you're right. I did this before putting in the error check, as a hedge in case the decision was not to error. Will fix. 

`addWordpressInstanceUrlAndVerify` will be false for all login requests now. We don't want to set referrer to wordpress for existing accounts.

Reply:
Perfect, thank you. Also great you chose the object param so we dont just have stray `false` and `true`s!